src/grading.py

```diff
@@ -41,12 +41,51 @@
 # from __future__ import unicode_literals # do not really fix path pb
 import sys
 from lxml import etree
-from PythonMagick import Image
+# from PythonMagick import Image
 import base64
-
+import os
 # ======================================================================
 # fonction pour le traitement des fichiers images
 # ======================================================================
+class ImageCustom:
+    def __init__(self,fileIn=None,fileOut=None):
+        self.pythonmagick = self.loadImageTool()
+        if fileIn is not None and fileOut is not None:
+            self.convertImage(fileIn,fileOut)
+
+    def loadImageTool(self):
+        """
+        chargement pythonmagick si disponible
+        """
+        loadok = False
+        try:
+            global Image
```

Comment:
we could avoid global here

Reply:
I am not sure... I had an error without the global declaration

Reply:
I don't know but I had an error without it